idx/cassandra/cassandra.go

```diff
@@ -266,8 +266,14 @@ func (c *CasIdx) AddOrUpdate(data *schema.MetricData, partition int32) {
 	}
 
 	if updateIdx {
+		// set the new metricdef based on the data, however if this data will go back in time
+		// it'll be rejected data, and we should stick to the higher LastUpdate we already had.
 		def := schema.MetricDefinitionFromMetricData(data)
+		if inMemory && existing.LastUpdate > def.LastUpdate {
```

Comment:
that's a good suggestion. though note that right now there's an important metric `tank.metrics_too_old` being incremented in the AggMetric.Add code. it's important we communicate this back to the user. i suppose we could increment that metric from here and then do as you say. (this would also bypass the usage tracking for these points - I think we should account for all points even if we reject them - though IIRC we plan to move the usage tracking to tsdb-gw so I think that's ok)